src/Gallery/GalleryItemSettings.tsx

```diff
@@ -54,9 +66,15 @@ export const GalleryItemSettings: FC<GalleryItemSettingsProps> = ({ use, selecte
             </div>
           ) : (
             <>
-              <p className="whppt-gallery__settings__title">{item.fileInfo?.originalname || ''}</p>
+              <div className="whppt-gallery-settings__header">
+                {item.fileInfo && <p className="whppt-gallery-settings__title">{formatFileName(item.fileInfo.originalname)}</p>}
+
+                <button className="whppt-gallery-settings__icon" onClick={() => setSettingsOpen(false)}>
+                  <WhpptIcon is="close" />
```

Comment:
I think Bens request was not so much for the settings panel in the gallery to close but for the whole gallery settings to close without having to click use.

Reply:
Discussed this with @BGillyVII  at the moment of making these changes.So the idea was that the `ImageEditor` was closing if we closed the gallery. I mafe the fix for that on the Nav actions. 
To close the gallery (without clicking "Use") the user can click on the nav icon again, This is actually how all screens behave ATM. If you think we should change it on that level, I can add to this branch too.

Reply:
OK lets go with that for now. I think the UX is not great though. Lets handle it later.

Reply:
I logged a seperate Asana ticket to do a ux review with Joey and then we can implement any feedback